packages/xo-web/src/xo-app/backup-ng/index.js

```diff
@@ -265,7 +265,7 @@ class JobsTable extends React.Component {
                   )}
                 </Li>
               )}
-              {compression !== '' && (
+              {compression !== '' && mode === 'full' && (
```

Comment:
IMHO we should mutualize the list of settings for each backup types between the display and the edition.

Otherwise there is a very high probability that we will again forget to update one or the other.